frontend/package.json

```diff
@@ -97,6 +97,7 @@
       "jest-dom"
     ],
     "rules": {
+      "@typescript-eslint/no-unused-vars": ["error"],
```

Comment:
⚠️ added this extra rule that is recommended so we don't get no unused var errors for types
https://stackoverflow.com/questions/67411154/eslint-no-unused-vars-on-function-parameters-within-typescript-types-interfa
https://typescript-eslint.io/rules/no-unused-vars/